bouncer/src/repo/core/model/bson/repo_node_mesh.h

```diff
@@ -230,16 +218,45 @@ namespace repo {
 
 			private:
 				/**
-				* Given a mesh mapping, convert it into a bson object
-				* @param mapping the mapping to convert
-				* @return return a bson object containing the mapping
+				* Retrieve a vector of faces (serialised) from the bson object
+				*/
+				std::vector<uint32_t> getFacesSerialized() const;
+			};
+
+			class REPO_API_EXPORT SupermeshNode : public MeshNode
```

Comment:
I think it'll be clearer to have this as a separate file (also the cpp)

Reply:
Done!